src/lineage/entrypoint.py

```diff
@@ -314,7 +319,14 @@ def main() -> None:
             continue
         logging.info(f"Lineage configuration found for {repo.full_name}")
         logging.info(f"Cloning repository: {repo.clone_url}")
-        run([GIT, "clone", repo.clone_url, repo.full_name])
+        # The failure of a repository to clone should not cause the entire
+        # run to fail.
```

Comment:
```suggestion
        # The failure of a repository to clone should not cause the entire
        # run to fail.  See related comment above in the body of the 
        # get_repo_list() function.
        #
        # See also issue #33 in this repository:
        # https://github.com/cisagov/action-lineage/issues/33
```

Reply:
Good call. Please see 3d7c955 as I had to make some edits to your suggestion.